include/srdfdom/model.h

```diff
@@ -41,8 +41,7 @@
 #include <string>
 #include <vector>
 #include <utility>
-#include <urdf_model/model.h>
-#include <srdfdom/urdfdom_compatibility.h>
+#include <urdf/model.h> // TODO: replace with urdf_model/types.h in Lunar
```

Comment:
I tried replacing it on a Ubuntu Zesty system and got an error:

> /root/ros/ws_catkin/src/srdfdom/include/srdfdom/model.h:66:28: error: 'ModelInterface' in namespace 'urdf' does not name a type

Which I have not looked into